python/doc/theory/probabilistic_modeling/dickey_fuller.rst

```diff
@@ -4,34 +4,42 @@ Dickey-Fuller stationarity test
 ===============================
 
 The Dickey-Fuller test checks the stationarity of a scalar time series using one time series.
-It assumes that the :math:`X: \Omega \times \cD \rightarrow \Rset` process with :math:`\cD \in \Rset`, discretized on the time grid :math:`(t_0, \dots, t_{N-1})` writes:
+It assumes that the :math:`X: \Omega \times \cD \rightarrow \Rset` process with
+:math:`\cD \in \Rset`, discretized on the time grid :math:`(t_0, \dots, t_{\sampleSize-1})` writes:
 
 .. math::
     :label: DFmodel
 
     X_t = a + bt + \rho X_{t-1} + \varepsilon_{t}
 
-where :math:`\rho > 0` and where :math:`a` or :math:`b` or both :math:`(a,b)` can be assumed to be equal to 0.
+where :math:`\rho > 0` and where :math:`a` or :math:`b` or both :math:`(a,b)` can be assumed
+to be equal to 0.
 
 The Dickey-Fuller test checks whether the random perturbation at time :math:`t` vanishes with time.
 
 When :math:`a \neq 0` and :math:`b=0`, the model :eq:`DFmodel` is said to have a *drift*.
 When :math:`a = 0` and :math:`b \neq 0`, the model :eq:`DFmodel` is said to have a *linear trend*.
 
-In the model :eq:`DFmodel`, the only way to have stochastic non stationarity is to have :math:`\rho = 1` (if :math:`\rho > 1`,
+In the model :eq:`DFmodel`, the only way to have stochastic non stationarity is to have
+:math:`\rho = 1` (if :math:`\rho > 1`,
 then the process diverges with time which is readily seen in the data).
-In the general case, the Dickey-Fuller test is a unit root test to detect whether :math:`\rho=1` against :math:`\rho < 1`:
+In the general case, the Dickey-Fuller test is a unit root test to detect whether
+:math:`\rho=1` against :math:`\rho < 1`:
```

Comment:
I do not understand what you mean. I propose to review this file in another pR. This is not the sacop of this PR.

Reply:
I agree this is beyond the scope of this PR. I was just wondering why the case where $\rho < 1$ is not included in the alternate hypothesis. Intuitively, I would have assumed that if the null hypothesis is $\rho=1$, then the alternate hypothesis should be $\rho \neq 1$ rather than $\rho > 1$.